server/app/main.py

```diff
@@ -26,7 +25,7 @@ async def get_status(request):
     import random
 
     payload = {
-        "node_identifier": "kabuto",
+        "sensor_identifier": "kabuto",
         "timestamp": utils.timestamp(),
         "values": {"value": random.randint(0, 2**10)},
     }
```

Comment:
Why do you perform this send operation on the get-status route? Just curious.

Reply:
Testing, I'll remove that soon.